assets/app/view/game/auto.rb

```diff
@@ -39,10 +39,10 @@ def render_content
           }.freeze
 
           if !(available = @game.available_programmed_actions).empty?
-            enabled = @game.programmed_actions[sender]
             available.each do |type|
+              enabled = @game.programmed_actions[sender].find { |a| a.is_a?(type) }
               if (method = types[type])
-                settings = enabled if enabled.is_a?(type)
+                settings = enabled if enabled
                 children.concat(method.call(settings))
```

Comment:
I wonder why we call this if settings is nil.

Reply:
To show the "Enable" button (instead of the "Disable" button).